src/services/authentication.js

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios';
+
+const API_HOST = process.env.REACT_APP_API_HOST || 'localhost'
+const API_URL = `http://${API_HOST}/api`

```

Comment:
In production, API URL might be using https so maybe the HTTP protocol should be included with API_HOST env variable instead.

Reply:
I agree with it and changed it!